joint_state_controller:
  type: "joint_state_controller/JointStateController"
  publish_rate: 50

head_controller:
  type: "position_controllers/JointPositionController"
  joint: base_to_head

front_right_arm_controller:
  type: "velocity_controllers/JointVelocityController"
  joint: base_to_front_right_arm

front_left_arm_controller:
  type: "velocity_controllers/JointVelocityController"
  joint: base_to_front_left_arm
  
back_right_arm_controller:
  type: "velocity_controllers/JointVelocityController"
  joint: base_to_back_right_arm
  
back_left_arm_controller:
  type: "velocity_controllers/JointVelocityController"
  joint: base_to_back_left_arm

front_right_arm_controller_turn:
  type: "position_controllers/JointPositionController"
  joint: base_to_front_right_arm_turn

front_left_arm_controller_turn:
  type: "position_controllers/JointPositionController"
  joint: base_to_front_left_arm_turn
  
back_right_arm_controller_turn:
  type: "position_controllers/JointPositionController"
  joint: base_to_back_right_arm_turn
  
back_left_arm_controller_turn:
  type: "position_controllers/JointPositionController"
  joint: base_to_back_left_arm_turn
